package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "server",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./server/bin/www",
+    "dev": "nodemon ./server/bin/www",
+    "dev:both": "cd client && npm start && cd .. && nodemon ./server/bin/www",
+    "test": "jest",
+    "client:init": "cd client && npm i",
+    "client:dev": "cd client && npm start",
+    "client:build": "cd client && npm run build"
+  },
+  "dependencies": {
+    "bcryptjs": "^2.4.3",
+    "body-parser": "^1.18.3",
+    "cookie-parser": "~1.4.3",
+    "debug": "~2.6.9",
+    "env2": "^2.2.2",
+    "express": "~4.16.0",
+    "http-errors": "~1.6.2",
+    "jsonwebtoken": "^8.3.0",
+    "morgan": "~1.9.0",
+    "pg": "^7.6.1",
+    "pug": "2.0.0-beta11",
```

Comment:
No need for this module.
it used for view engine in express and we didn't set up it

Reply:
Cool I've removed this now.